llvm/test/tools/sycl-post-link/virtual-functions/module-cleanup-comdat.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: sycl-post-link -split=auto -S < %s -o %t.table
+;
+; Virtual functions cleanup drops their bodies from some of device images
+; turning them into declarations, but declarations can't have "comdat" thing
```

Comment:
```suggestion
; turning them into declarations, but declarations can't have "comdat"
```

Reply:
Applied in 9bef3a14ff2c8083378a19d7dd5a5c77d7ef1114